### ISSUE TITLE
[feat] New rule `no-explict-spread-props`

### ISSUE THREAD

**Issue by SukkaW** [Rel1cx/eslint-react#503]
### Describe the problem

```jsx
    <a
      className={styles.link}
      {...{
        href: props.href,
        rel: 'noopener noreferrer',
        target: '_blank',
      }}
      {...props}
    >
      {label}
    </a>
```

### Describe the solution you'd like

This kind of code should be forbidden and should be auto fixed to

```jsx
    <a
      className={styles.link}
      href={props.href}
      rel="noopener noreferrer"
      target="_blank"
      {...props}
    >
      {label}
    </a>
```

### Additional context

This should be allowed:

```jsx
    <Comp
      {...(
        props.a ? { b: 'c' } : {}
      )}
      {...props}
    />
```
